app/templates/weekview.html

```diff
@@ -12,12 +12,17 @@
     <body>
         <div id="week-view">
             <div id="week-schedule" class="d-flex justify-content-between">
-                {% for day, dayview, events_and_attr in week %}
+                {% for day, dayview, events_and_attr, current_time_and_attrs, all_day_events in week %}
                 <div class="day-weekview m-0 flex-fill border-start" style="width: 100%;">
-                        <div class="day-name sticky-top">{{ day.strftime('%A').upper()[:3] }}</div>
+                    <div class="day-name sticky-top">{{ day.strftime('%a').upper()}}
+                        {% for event in all_day_events %}
+                        <a href={{ url_for('eventview', event_id=event.id) }} title="See full event" class="text-danger">{{ event.title }}</a>
+                        {% endfor %}
+                    </div>
                         {% set month = day.month %}
                         {% set day = day.day %}
-                        {% set events = events_and_attr%}
+                        {% set events = events_and_attr %}
+                        {% set current_time = current_time_and_attrs %}
```

Comment:
Why not using `events_and_attr` as is?

Reply:
What does it mean as is?

Reply:
Why should we set a new variable called `events`?

Reply:
If I understand it correctly, it's not about re-setting it rather than just sending it to 'week16\calendar\app\templates\calendar_day_view.html'